### ISSUE TITLE
[InputTextMixin] the method _getEntity() doesn't work du to a change in getValue method

### ISSUE THREAD

**Issue by JRLK** [KleeGroup/focus-components#278]
In src/common/input/text/index.js for the last version of Focus-component
```javascript
    getValue: function getValue() {
        return this.props.unformatter(React.findDOMNode(this.refs.inputText).value);
    },
```

In common/input/text/index.je for Focus-component v0.4.0
```javascript
    getValue: function getInputTextValue() {
        return this.props.unformatter(React.findDOMNode(this).value);
    },
```

Call _getEntity() return this error  when called in a changeHandler on a checkbox:
```javascript
Uncaught TypeError: Cannot read property 'value' of null
```

**Comment by Tommass** [comment]
corrected in v0.6.0

**Comment by Tommass** [comment]
https://github.com/KleeGroup/focus-components/releases/tag/v0.6.1
